docs/en/stock-management/accounting/index.md

```diff
@@ -0,0 +1,163 @@
+&raquo; [Home](../../index.md) / [Inventory & Stock Management](../index.md) / Stock Management & Accounting
+
+# Stock Management & Accounting
+Stock management in BHIMA can be a standalone package or with automated accounting features.  The automated
+accounting features are enabled in the [Stock Settings](../stock.settings.md#individual-stock-settings) module under
+the "Enable Automatic Stock Accounting" setting.  It is on by default.
+
+BHIMA behaves differently depending on the accounting standard being followed.  For completeness, below we'll present
+the two accounting strategies and discuss how BHIMA automates each.  We'll present an overview, then the purchasing procedure,
+then the consumption of medicines.
+
+## OHADA Stock Accounting
+
+### Overview
+In OHADA accounting, purchases are modeled as an expense.  When goods are delivered to the enterprise, the expense is offset
+with a credit to bring the sum of the expense back to zero.  Finally, the consumption of medicines decreases the value of goods
+in stock and creates an expense for the institution.
+
+These are the accounts used in the OHADA examples below:
+
```

Comment:
I think it is worth noting briefly here that these are families of accounts (that is my understanding).  In the OHADA section, you could go into more detail, eg, why different sub-accounts are used, etc.

Reply:
Done.

I don't know that there is any rhyme or reason to the OHADA sub accounts and they sometimes change, so I'd prefer not to name specifics here.